webapp-django/crashstats/sources/views.py

```diff
@@ -50,5 +50,5 @@ def highlight_url(request):
     else:
         lexer = get_lexer_for_filename(filename)
 
-    formatter = HtmlFormatter(full=True, title=parsed.path, linenos='table', lineanchors='1')
+    formatter = HtmlFormatter(full=True, title=parsed.path, linenos='table', lineanchors='L')
```

Comment:
The lineanchor here was set to `1` (one), but should have been `l` (lower-case ell) which is my fault. @luser, after spending some time noticing the problem and then hunting for the cause, suggested that we change it from an `l` (lower-case ell) to an `L` (upper-case ell) which I thought was a pretty swell idea.